.github/workflows/npm-release.yml

```
@@ -0,0 +1,58 @@
+name: release-node-image
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)